src/instances/RaidInstance.ts

```diff
@@ -170,44 +174,31 @@ export class RaidInstance {
     private readonly _raidSection: ISectionInfo;
     // Number of people that can get early location through Nitro.
     private readonly _numNitroEarlyLoc: number;
-
+    // All essential options (options that give early location). Equivalent to _afkCheckButtons but as raw data
     // Nonessential reactions. These are reactions that don't give any perks. More can be added at any point.
     private readonly _nonEssentialReactions: EmojiIdentifierResolvable[];
-
+    // A collection that contains the IAfkCheckReaction.mapKey as the key and the members with the corresponding
     // Buttons to display on the AFK check. These should only contain essential buttons.
     private readonly _afkCheckButtons: MessageButton[];
-    // All essential options (options that give early location). Equivalent to _afkCheckButtons but as raw data
+    // A collection that deals with *general* (Nitro, Patreon, etc.) early location. The key is the mapKey and the
     // instead of buttons. The key is the mapping key.
     private readonly _allEssentialOptions: Collection<string, ReactionInfoMore>;
-    // A collection that contains the IAfkCheckReaction.mapKey as the key and the members with the corresponding
     // item as the value.
```

Comment:
Same issue with comment order.  More stacked single-line comments that dont appear to match what they describe.  If these are intended to be multi-line comments, would make them so. and double check the wording.

Same for HeadcountInstance

Reply:
should be acknowledged